chat-server/src/routes/conversations/rateMessage.ts

```diff
@@ -0,0 +1,69 @@
+import { ObjectId } from "mongodb";
+import {
+  ConversationsServiceInterface,
+  Message,
+} from "../../services/conversations";
+import {
+  Request as ExpressRequest,
+  Response as ExpressResponse,
+  NextFunction,
+} from "express";
+
+interface RatingRequest extends ExpressRequest {
+  params: {
+    conversationId: string;
+    messageId: string;
+  };
+  body: {
+    conversation: Message[];
+    id: string;
+    ip_address: string;
+    message_index: number;
+    rating: boolean;
+  };
```

Comment:
```suggestion
  body: {
    rating: boolean;
  };
```

We only need `rating` here